lib/register.php

```diff
@@ -317,7 +317,19 @@ function gutenberg_can_edit_post_type( $post_type ) {
  */
 function gutenberg_post_has_blocks( $post ) {
 	$post = get_post( $post );
-	return $post && strpos( $post->post_content, '<!-- wp:' ) !== false;
+	return $post && content_has_gutenberg_blocks( $post->post_content );
```

Comment:
We should be consistent here in how we're treating "Gutenberg" naming; for the time-being it's more a prefix to namespace the functions, so I would expect this to be `gutenberg_content_has_blocks`, and maybe eventually simply `content_has_blocks` (we don't want Gutenberg nomenclature to stick around forever).

Reply:
yeh, I was trying to namespace it - but then it looked like it was checking gutenberg content, so tried to make it more clear. Switched back to just `content_has_blocks` which is what it was before.

:+1:

Reply:
I think since we changed to prefix the "post has blocks" function, we should do the same with this one: `gutenberg_content_has_blocks`

Also, there were unit tests for this before I believe? Can those be restored?